WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -820,6 +820,9 @@ public void onActionResult(CommentActionResult result) {
                         ToastUtils.showToast(getActivity(), getString(R.string.note_reply_successful));
                         mEditReply.setText(null);
                         mEditReply.getAutoSaveTextHelper().clearSavedText(mEditReply);
+
+                        // approve the comment
+                        moderateComment(CommentStatus.APPROVED);
```

Comment:
You're right. Do you think it's enough to just check for the approved status or should there be some more elaborate logic for other states? The thing is in wp-admin you can't reply to trashed or spam comments but you can on mobile.